src/components/MainPage.js

```diff
@@ -59,48 +56,70 @@ type Props = {
 
 type State = {
   inputValue: number | string,
-  favorites: Array<number>,
+  activePage: number,
+  favorites: string,
 };
 
 class MainPage extends Component<Props, State> {
   componentDidMount() {
-    this.props.getMovies.movies();
+    this.props.getMovies.movies(null, 1);
+    this.setState({ activePage: 1 });
+    localStorage.setItem('favorites', 'init');
   }
 
   handleSearchChange = (event: SyntheticInputEvent<HTMLInputElement>) => {
     this.setState({ inputValue: event.target.value });
+    this.setState({ activePage: 1 });
   };
 
   handleSubmit = (event: SyntheticInputEvent<HTMLInputElement>) => {
     event.preventDefault();
-    this.props.getMovies.movies(this.state.inputValue);
+    this.props.getMovies.movies(this.state.inputValue, this.state.activePage);
   };
 
-  //TODO: add functionality to handle list of favorites movies and bottom navigation
-  handleBottomNavigationChange = (
-    event: SyntheticInputEvent<HTMLInputElement>
-  ) => {
-    event.preventDefault();
-    console.log('handleBottomNavigationChange');
-  };
-
-  openListOfFavorites = (event: SyntheticInputEvent<HTMLInputElement>) => {
-    event.preventDefault();
-    console.log('openListOfFavorites');
+  handlePageChange = pageNumber => {
+    const value = this.state.inputValue;
+    if (value) {

```

Comment:
Can be replaced with ternary, something like

`const value = this.state.currentValue || null`

Reply:
Done